README.md

```diff
@@ -64,6 +66,8 @@ If using any OS other than Fedora/Rhel (i.e., Mac, Ubuntu Linux):
 
 `npx playwright test test_navigation.test.ts -g "User can navigate to the workspaces page via the menu"`
 
+Note: You can run all of the above to the stage env setup directly without adding the `-e HTTPS_PROXY=$RH_PROXY_URL` var in the proxy command 
```

Comment:
I don't think this is true, the proxy container needs the `HTTPS_PROXY` setup to work against stage 🤔